packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -344,6 +344,8 @@ abstract class ScrollView extends StatelessWidget {
 /// This sample code shows a scroll view that contains a flexible pinned app
 /// bar, a grid, and an infinite list.
 ///
+/// {@animation 400 376 https://flutter.github.io/assets-for-api-docs/assets/widgets/custom_scroll_view.mp4}
```

Comment:
Animations are not supported inside of the `{@tool sample}`, so let's move this just above that around line 341 and have the description refer to the animation above.

Reply:
Also, check the sample below to update the text to match the new diagram.